aspnetcore/fundamentals/localization/sample/Views/Shared/_SelectLanguagePartial.cshtml

```diff
@@ -19,6 +19,7 @@
           asp-action="SetLanguage" asp-route-returnUrl="@Context.Request.Path" 
           method="post" class="form-horizontal" role="form">
         @Localizer["Language:"] <select name="culture"
+          onchange="this.form.submit();"
```

Comment:
Because when I change the language, the page doesn't Post any data (Home/SetLanguage). With this behaviour the Cookies is not set with the selected language.